Source/WebCore/page/Performance.cpp

```diff
@@ -489,31 +475,29 @@ ExceptionOr<Ref<PerformanceMeasure>> Performance::measure(JSC::JSGlobalObject& g
     if (measure.hasException())
         return measure.releaseException();
 
-    if (isSignpostEnabled()) {
```

Comment:
Have all stake holders reached an agreement on this? I would assume doing signpost has some overhead.